agentstack/tools/neon.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "neon",
+    "package": "poetry add neon-api",
+    "env": "NEON_API_KEY=...",
+    "tools": ["neon"],
```

Comment:
`tools` references the variable that are available in the tool implementation ie. neon_tool.py, so this needs to be:

`"tools": ["create_database", "execute_sql_ddl", "run_sql_query"],`